File: Arcade/Intro/C++/03_checkPalindrome.cpp
bool checkPalindrome(std::string inputString) {
        int j=inputString.size()-1;
        for(int i=0;i<inputString.size();i++){
              if(inputString[i]!=inputString[j]){
                     return false;
              }
              j--;
        }
        return true;
}

File: Arcade/Intro/JavaScript/03_checkPalindrome.js
function checkPalindrome(inputString) {
    var len = inputString.length;
    for (var i = 0; i < len/2; i++) {
        if (inputString.charAt(i) !== inputString.charAt(len-1-i)) {
            return false;
        }
    } return true;
}
